integration-tests/smallrye-stork-consul-registration/pom.xml

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-integration-tests-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>quarkus-integration-test-smallrye-stork-consul-registration</artifactId>
+    <name>Quarkus - Integration Tests - Smallrye Stork Automatic Consul Registration Prod Mode</name>
```

Comment:
If you want to test native coverage, you probably need to add it in https://github.com/quarkusio/quarkus/blob/main/.github/native-tests.json .